.gitignore

```diff
@@ -0,0 +1,126 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```

Comment:
I suggest don't use any template for git ignore, which who make you later forgot what you add into here, manually remove those you don't want would be better

Reply:
you are right.